hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/Store.java

```diff
@@ -126,6 +126,11 @@
    */
   int getStorefilesCount();
 
+  /**
+   * @return Reference count over store
+   */
+  int getStoreRefCount();
```

Comment:
So here we also expose this value to CP, not only through metrics?

Reply:
I think it would be good to have this in the interface to avoid some casting but is not required probably

Reply:
If you mean this code sniff
`        for (Store store : region.stores.values()) {`
`          tempNumStoreFiles += store.getStorefilesCount();`
`          tempStoreRefCount += store.getStoreRefCount();`
`          tempMemstoreSize += store.getMemStoreSize().getDataSize();`
`          tempStoreFileSize += store.getStorefilesSize();`
`          OptionalLong storeMaxStoreFileAge = store.getMaxStoreFileAge();`

Actually you can just change  the `for (Store store : region.stores.values()) {` to `for (HStore store : region.stores.values()) {`, as the `region` is a `HRegion`.